pym/portage/util/writeable_check.py

```diff
@@ -57,8 +57,22 @@ def linux_ro_checker(dir_list):
 				# there can be a variable number of fields
 				# to the left of the ' - ', after the attr's, so split it there
 				mount = line.split(' - ', 1)
-				_dir, attr1 = mount[0].split()[4:6]
-				attr2 = mount[1].split()[2]
+				try:
+					_dir, attr1 = mount[0].split()[4:6]
+				except ValueError:
+					# If it raises ValueError we can simply ignore the line.
+					continue
+				# check for situation with invalid entries for /home and /root in /proc/self/mountinfo
+				# root path is missing sometimes on WSL
+				# for example: 16 1 0:16 / /root rw,noatime - lxfs  rw
+				if len(mount) > 1:
+					try:
+						attr2 = mount[1].split()[2]
+					except IndexError:
+						try:
+							attr2 = mount[1].split()[1]
+						except IndexError:
+							attr2 = mount[1]
```

Comment:
In this case it would be a format that we've never seen before, so I think we should ignore the whole line in this case.